app/Models/AlertRule.php

```diff
@@ -52,7 +52,7 @@ public function scopeIsActive($query)
     {
         return $query->enabled()
             ->join('alerts', 'alerts.rule_id', 'alert_rules.id')
-            ->where('alerts.state', 1);
+            ->whereIn('alerts.state', [1, 3, 4]);
```

Comment:
I noticed you use whereNotIn in the legacy index and whereIn here.  Not an issue just an inconsistency.

Reply:
Corrected.